src/community/infix-eng.ts

```diff
@@ -0,0 +1,14 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import { toSubscript } from "../utils";
+
+export class InfixEngineeringNotation extends AbstractInfixNotation {
+  public readonly name = "Infix engineering";
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'name' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)